README.md

```diff
@@ -26,14 +18,35 @@ Conjur provides:
 
 * try Conjur: [Start Here](https://try.conjur.org)
 * [support](https://try.conjur.org/support.html)
-* [API Documentation][api-doc]
+* [API Documentation][api]
+
+[![Join Conjur Slack](https://img.shields.io/badge/slack-open-e01563.svg)][slack]
+[![Conjur API Ruby Gem](https://img.shields.io/gem/dt/conjur-api.svg?label=ruby%20gem%20downloads)][gem]
+[![Docker Pulls](https://img.shields.io/docker/pulls/cyberark/conjur.svg)][docker]
```

Comment:
Does this go to the DockerHub? We'll want shields for Quay and DockerHub, Quay first since it gives better detail. Pull #s aren't especially important to feature, i don't think.

Reply:
Switched to using a Quay badge